client/interledger/js/components/account_detail.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import classnames from 'classnames';
+import { Col } from 'react-bootstrap/lib';
+
+import Assets from './assets';
+
+
+const AccountDetail = React.createClass({
+    propTypes: {
+        account: React.PropTypes.object,
+        activeAccount: React.PropTypes.object,
+        assetList: React.PropTypes.object,
+        handleClick: React.PropTypes.func
+    },
+
+    render() {
+        const {
+            account,
+            activeAccount,
+            assetList,
+            handleClick
+        } = this.props;
+        
+        if (account) {
+            const assetListForAccount = assetList[account.vk];
+            return (
+                <Col
+                    className={classnames({ 'active': activeAccount === account })}
+                    md={6} lg={4} xl={3}
```

Comment:
Styleguide:

Every property on it's own new line.
But I'd say in this case it's fine.